packages/frontend/web-ui/src/common/models/DisplayDialog.ts

```diff
@@ -0,0 +1,5 @@
+export interface DisplayDialog {
```

Comment:
maybe `UseDisplayDialogResult` makes more sense given the names we currently use